frontend/src/routes/(authenticated)/project/[project_code]/AddOrganization.svelte

```diff
@@ -18,32 +21,36 @@
 
   async function openModal(): Promise<void> {
     await formModal.open(async () => {
-      await _addProjectToOrg({
+      const { error } = await _addProjectToOrg({
         projectId,
         orgId: $form.orgId,
       })
+      if (error?.byType('NotFoundError')) {
+        if (error.message === 'Organization not found') return $t('project_page.add_org.org_not_found');
+      }
+      if (error?.byType('NotFoundError')) {
```

Comment:
These two if blocks would be better as a single `if (error?.byType('NotFoundError'))` block.